src/lib/db/schema.ts

```diff
@@ -120,3 +139,44 @@ export const bookingSessionRelations = relations(bookings, ({ one }) => ({
 export const userSessionRelations = relations(users, ({ many }) => ({
   bookings: many(bookings),
 }));
+
+export const semesters = pgTable("semester", {
+  id: serial("id").primaryKey(),
+  startDate: timestamp("startDate", { mode: "date" }).notNull(),
+  endDate: timestamp("endDate", { mode: "date" }).notNull(),
+  breakStart: timestamp("breakStart", { mode: "date" }).notNull(),
+  breakEnd: timestamp("breakEnd", { mode: "date" }).notNull(),
+  bookingDay: timestamp("bookingDay", { mode: "date" }).notNull(),
+  bookingOpenTime: timestamp("bookingOpenTime", { mode: "date" }).notNull(),
```

Comment:
this represents a time i.e 9:00, 21:00 etc. there should be a postgres type that represents that. refer to the drizzle docs